media/windows/ECC-Ecobee/ECC Pycobee Data.py

```diff
@@ -268,7 +287,7 @@
                                'hierarchy_set': 'HierarchySet', 'hierarchy_user': 'HierarchyUser',
                                'limit': 'LimitSetting', 'outputs': 'Output', 'page': 'Page',
                                'remote_sensors': 'RemoteSensor', 'capability': 'RemoteSensorCapability',
-#                               'reminders': 'ThermostatReminder2', 'runtime_sensor_metadata': 'RuntimeSensorMetadata',
+                               #                               'reminders': 'ThermostatReminder2', 'runtime_sensor_metadata': 'RuntimeSensorMetadata',
```

Comment:
Tiny nit: don't know if you meant to do this.

Reply:
If you mean the large indent on the commented out line...sheesh, no!  Not sure how that happened but cleaned it up for the next iteration...